src/RefractAST.cc

```diff
@@ -315,36 +327,82 @@ namespace drafter
                 }
             }
 
-            if (IElement* e = SimplifyRefractContainer(samples)) {
-                element->attributes["sample"] = e;
-            }
+        }
 
-            if (IElement* e = SimplifyRefractContainer(defaults)) {
-                element->attributes["default"] = e;
-            }
+        if (IElement* e = SimplifyRefractContainer(samples)) {
+            element->attributes["sample"] = e;
         }
 
-        if(!description.empty()) {
-            element->meta["description"] = IElement::Create(description);
+        if (IElement* e = SimplifyRefractContainer(defaults)) {
+            element->attributes["default"] = e;
         }
 
+
         return element;
     }
 
     template <typename T>
-    refract::IElement* RefractElementFromProperty(const mson::PropertyMember& property)
+    refract::MemberElement* RefractElementFromProperty(const mson::PropertyMember& property)
     {
-        refract::IElement* element = RefractElementFromValue<T>(property);
-        if (!property.name.literal.empty()) {
-            element->meta["name"] = refract::IElement::Create(property.name.literal);
+        //if (property.name.literal.empty()) {
```

Comment:
commented -out code without any explanation. Please either add a comment explaining why this is commented out and until when or remove it completely